modules/solid_mechanics/test/tests/neml2/explicit_dynamics/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  issues = '#30850'
+  design = 'NEML2/index.md NEML2Action.md'
+  [explicit_dynamics]
```

Comment:
Is there some way of testing that the results from neml2 match regular moose? Incase it diverges somehow in the future.